reconcile/gql_definitions/terraform_cloudflare_dns/terraform_cloudflare_zones.gql

```diff
@@ -0,0 +1,64 @@
+# qenerate: plugin=pydantic_v1
+
+query CloudflareDnsZone {
+  zones: cloudflare_dns_zone_v1 {
+    identifier
+    zone
+    account {
+      name
+      type
+      description
+      providerVersion
+      enforceTwofactor
+      apiCredentials {
+        path
+        field
+        format
+        version
+      }
+      terraformStateAccount {
+        name
+        consoleUrl
+        terraformUsername
+        automationToken {
+          path
+          field
```

Comment:
Same as above: `... VaultSecret`

Reply:
It will also make the tests easier, as you will not have to import 2 different `VaultSecret` classes anylonger for the tests.